tests/t900-libparanumal.c

```diff
@@ -0,0 +1,38 @@
+/// @file
+/// Test creation and destruction of a CEED object
+/// \test Test creation and destruction of a CEED object
+#include <ceed.h>
+
+int main(int argc, char **argv) {
+  Ceed ceed;
```

Comment:
This should eventually become a t520 series test - if this is a basic elliptic PDE on hexes, then all backends should be able to impliment it from the gallery

I am confused as to where the data for the vectors is coming from - they are not ever created!

Also, the restrictions should be destroyed at the end of the test.

Reply:
This test is not full, those fields will have to be initialized indeed.